traffic_monitor/threadsafe/cacheavailablestatus.go

```diff
@@ -20,8 +20,8 @@ package threadsafe
  */
 
 import (
-	"github.com/apache/trafficcontrol/traffic_monitor/cache"
 	"github.com/apache/trafficcontrol/lib/go-tc"
+	"github.com/apache/trafficcontrol/traffic_monitor/cache"
 	"sync"
 )
```

Comment:
since we're reordering anyway,  can you move the "sync" to the top with a blank line after it?  go fmt will keep the std library separate from the rest that way -- same with the file below..

Reply:
done!